kyo-test/shared/src/main/scala/kyo/test/interop/zio.scala

```diff
@@ -0,0 +1,15 @@
+package kyo.test.interop
+
+import kyo.*
+import zio.Trace
+import zio.ZIO
+
+extension (capture: ZIO.type)
+    private[kyo] def fromKyoFiber[A](fiber: Fiber[A])(using Flat[A], Trace): ZIO[Any, Throwable, A] =
+        ZIO.asyncInterrupt[Any, Throwable, A] { cb =>
+            val async: ZIO[Any, Throwable, A] = ZIO.fromFuture(_ => IOs.run(fiber.toFuture))
```

Comment:
We could probably replace `fromFuture`, but I'm not too worried about the performance or cost given this should only be used in tests.